### DISCUSSION TITLE
How to set advanced options via API, I haven't found a way to do it.

### DISCUSSION THREAD

**Discussion by rafaelsanchezrd** [pschlan/cron-job.org#189] [Q&A]
Hello, is there a way to set advance options via API, I haven't found a way to do it.

<img width="565" alt="image" src="https://user-images.githubusercontent.com/5248544/221246135-ff57fc26-d9c3-4818-bd48-1b1d176aeb2f.png">

**Comment by pschlan** [comment]
Hi,

request method can be set via Job::requestMethod (https://docs.cron-job.org/rest-api.html#job)
The body can be set in DetailedJob::extendedData::body (https://docs.cron-job.org/rest-api.html#jobextendeddata)
